frontend/public/locales/en/translation.json

```diff
@@ -153,6 +153,7 @@
     "Automations": "Automations",
     "Autoscaling": "Autoscaling",
     "Available cluster types": "Available cluster types",
+    "Azure Resource Groups are logical collections of virtual machines, storage accounts, virtual networks, web apps, databases, and/or database servers. Typically, users group related resources for an application, divided into groups for production and non-production.": "Azure Resource Groups are logical collections of virtual machines, storage accounts, virtual networks, web apps, databases, and/or database servers. Typically, users group related resources for an application, divided into groups for production and non-production.",
```

Comment:
Why is this repeated?

Reply:
this is an autogenerated code, it is a key-value pair.